### PR TITLE
Raf debugging teal and examples

### PR DESCRIPTION
**Pull request by rfustino**

### CODE REVIEW

File: docs/features/asc1/.pages [hunk 1]
```diff
@@ -4,4 +4,5 @@ arrange:
    - stateful
    - stateless
    - teal 
+   - build.md
```
Comment (ejbaran) [inline docs/features/asc1/.pages:7]: Remove this.

File: docs/features/asc1/debugging.md [hunk 2]
```diff
@@ -1 +1,504 @@
-title: Debugging TEAL
\ No newline at end of file
+title: Smart Contract Debugging
+
+# Writing logic signature transaction for Debugging
```
Comment (ejbaran) [inline docs/features/asc1/debugging.md:3]: Capitalization

File: docs/features/asc1/debugging.md [hunk 3]
```diff
@@ -1 +1,504 @@
...
+The SDKs and goal command-line tool provide functionality to do a test run of a TEAL program. The goal command-line tool can be used with [stateless smart contracts.(/stateless/walkthrough.md)]
```
Comment (ejbaran) [inline docs/features/asc1/debugging.md:4]: [stateless smart contracts.(/stateless/walkthrough.md)] --> [stateless smart contracts](/stateless/walkthrough.md).

File: docs/features/asc1/.pages (hunk 1 above)
Comment (rfustino) [reply docs/features/asc1/.pages]: fixed

File: docs/features/asc1/debugging.md (hunk 2 above)
Comment (rfustino) [reply docs/features/asc1/debugging.md]: Better than new, fixed

File: docs/features/asc1/debugging.md (hunk 3 above)
Comment (rfustino) [reply docs/features/asc1/debugging.md]: fixed

**Comment by rfustino** [comment]
Removal of Writing Logic Signature section has been completed.

**Pull request merged**